template/go/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.9.7-alpine3.8 as builder
+FROM golang:1.10-alpine3.8 as builder
```

@alexellis should this be locked to a specific patch release for consistency of builds rather than the free floating `major.minor`.

```suggestion
FROM golang:1.10.4-alpine3.8 as builder
```